### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by miyer26**
Added the project proposal. This is the first draft, the link to the EDA needs to be added.

### CODE REVIEW

**Review by showcy — CHANGES_REQUESTED** [review]
Seems some grammar issues.
Change to:
"Each entry in the dataset corresponds to a single offender serving a two or more years long sentence. The details such as age, gender, and marital status at end of year are provided for each entry, and was retrieved from the Offender Management System (OMS)."

**Comment by miyer26** [comment]
Agreed, this change needs to be incorporated. As the EDA document is now on the repo, the link to that needs to be updated on the readme too.

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+
+The final evaluation will be completed by obtaining the p-value based on the effect size for our samples and comparing it to our significance level. If the p-value is smaller than the significance level, we can claim that there is enough statistical evidence to reject the null hypothesis. The null-distribution and test-statistic will be visualized as presented as a figure, and the p-value and effect size in a table. Whilst evaluating our findings, we will need to consider the power of the test based on the sample size and how large of an effect we observed. The preliminary EDA can be found in this [file] (link to EDA file in repo).
+
+## Usage
+
+## Dependencies
+
+## License
```
Comment (AraiYuno) [inline README.md:23]: Let's add our MIT license!

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+
+The final evaluation will be completed by obtaining the p-value based on the effect size for our samples and comparing it to our significance level. If the p-value is smaller than the significance level, we can claim that there is enough statistical evidence to reject the null hypothesis. The null-distribution and test-statistic will be visualized as presented as a figure, and the p-value and effect size in a table. Whilst evaluating our findings, we will need to consider the power of the test based on the sample size and how large of an effect we observed. The preliminary EDA can be found in this [file] (link to EDA file in repo).
+
+## Usage
```
Comment (Radascript) [inline README.md:19]: don't need this yet

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+
+## Dependencies
```
Comment (Radascript) [inline README.md:21]: will be added in later milestones

**Pull request merged**